scripts/measure_tool_list.py

```
@@ -0,0 +1,98 @@
+    for spec in specs:
+        desc_chars = len(spec.description)
+        params_chars = len(json.dumps(spec.parameters))
+        total = desc_chars + params_chars
+        rows.append((spec.name, desc_chars, params_chars, total))
+    rows.sort(key=lambda r: -r[3])
+
+    total_desc = sum(r[1] for r in rows)
+    total_params = sum(r[2] for r in rows)
+    total_combined = total_desc + total_params
+
+    print(f"=== Tool list size summary ===")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)